platform/innovium/docker-syncd-invm-rpc/Dockerfile.j2

```diff
@@ -15,19 +15,23 @@ RUN apt-get purge -y syncd
 RUN apt-get update \
  && apt-get -y install  \
     net-tools           \
-    python-pip          \
-    python-setuptools   \
+    python3-pip          \
+    python3-setuptools   \
     build-essential     \
     libssl-dev          \
     libffi-dev          \
-    python-dev          \
+    python3-dev          \
```

Comment:
Please add lines to uninstall the development packages afterwards.

Reply:
Thanks I have taken care in updated patch